interfaces/builtin/opengl.go

```diff
@@ -91,10 +94,11 @@ unix (bind,listen) type=seqpacket addr="@cuda-uvmfd-[0-9a-f]*",
 @{PROC}/driver/prl_vtg rw,
 
 # /sys/devices
-/sys/devices/pci[0-9a-f]*/**/config r,
-/sys/devices/pci[0-9a-f]*/**/revision r,
-/sys/devices/pci[0-9a-f]*/**/{,subsystem_}device r,
-/sys/devices/pci[0-9a-f]*/**/{,subsystem_}vendor r,
+/sys/devices/**pci[0-9a-f]*/**/config r,
```

Comment:
What is the ** for? what was the actual path you saw here?

Reply:
It is in the commit message:

/sys/devices/1003000.pcie-controller/pci0000:00/0000:00:02.0/*